### ISSUE TITLE
Can we release versions of these actions?

### ISSUE THREAD

**Issue by jbusecke** [ProjectPythia/cookbook-actions#112]
I find myself using these actions in many different projects, super helpful, thanks for maintaining them. 

I am currently often pinging to main, but that is problematic. Could we establish releases here so that I can pin them more easily in the future? 

Thank  you

**Comment by brian-rose** [comment]
Yes, we should be doing this. I think the main blocker holding us back from a more formal release has been the lack of testing, i.e. #6

**Comment by jbusecke** [comment]
Pinging this from #120. Once that is closed can we release something regardless? Happy to go with something like `0.1.0rc` to indicate this is not tested, but having a stable release is important from our end at LEAP.

**Comment by brian-rose** [comment]
Sure. I don't see any downside. I'll cut a release here.

**Comment by jbusecke** [comment]
Thank you!

**Comment by brian-rose** [comment]
Done, see https://github.com/ProjectPythia/cookbook-actions/releases/tag/v2024.10.29

**Comment by jbusecke** [comment]
Amazing!
